### ISSUE TITLE
[Improvement] Setup Wizard doesn't have a `Save`/`Continue` green button

### ISSUE THREAD

**Issue by wedebe** [openatv/enigma2#2703]
Some setup screens don't have descriptive text for what to do to progress the wizard. Pressing `OK` on the remote is ambiguous as the user could assume that this will change the currently-selected value. Now they find themselves at the next step, trying to find their way back to a setting they may have wanted to change.

**Comment by jbleyel** [comment]
This is a massive effort and not really needed imho.

**Comment by IanSav** [comment]
If you are in a single wizard script you can press EXIT to go back a step.

In the future I would like to make the wizard a single script so that users can navigate backwards through *all* the wizard steps.

**Comment by jbleyel** [comment]
But this is very low priority because the wizard will be started normally only once if you never had a receiver before.
In all other cases you have existing settings and you use fast restore.

I have personally have seen this wizard only once and this was 5 years ago.
